docs/changelog/fragments/2026-07-14-fix-djf-io-search-hint-vertical-centering.md

```
@@ -0,0 +1,7 @@
+### fix(djf.io): vertically center the search hotkey hint chip
```

markdownlint-cli2 (0.23.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)